### ISSUE TITLE
Support: taxii_push.py missing module

### ISSUE THREAD

**Issue by Tubbs-T** [MISP/MISP#10055]
### Support Questions

I am trying to push to a taxii server that I have started in my environment by using taxii_push.py for the GUI. The error I am running into at this point is below.

```Traceback (most recent call last):
  File "/var/www/MISP/app/files/scripts/taxii/taxii_push.py", line 9, in <module>
    import taxii2client
ModuleNotFoundError: No module named 'taxii2client'
```

My initial problem is similar to that in #9843 following the steps provided in that issue did not solve my problem. I installed all of the same packages that were recommended using ``` sudo ``` but I had to use the ```--break-system-packages``` to install.

Running the package does exist both within and outside my virtual environment

```
var/www/MISP/venv/bin$ pip freeze | grep taxii2
taxii2-client==2.3.0
```

For more context I was on the most recent version of MISP 2.4.199, Ubuntu 22.04 and was running into the same problem from #9843 and #4107 where this was the error when running the same script.

```
stix2.exceptions.ExtraPropertiesError: Unexpected properties for Identity: (interoperability)
```

I was able to solve this problem on the older version by following the recommendations provided in the issues and that solved my problem but after solving that is when I ran into this current one. I then updated Ubuntu to 24.04 and the subsequently updated MISP to the latest version and I still am experiencing this problem.

### MISP version

2.5.2

### Operating System

Ubuntu  

### Operating System version

24.04

### PHP version

8.3.6

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by mark00411** [comment]
Hey Tubbs, did you manage to resolve this issue? Running into the same issue exactly after updating to MISP 2.5 and Ubuntu 24.04. Thanks!

**Comment by Tubbs-T** [comment]
@mark00411 I was able to fix the problem, so I checked the venv that MISP was running I and I did not see the packages listed under that, so I installed the required packages using that venv instance. The other potential fix was to alter the `pyvenv.cfg` and change `include-system-site-packages` to `true` and the will let you install python modules using the `--break-system-packages`. I not sure if this is recommended since I do not know the security implications but it should get it working.
